tests/libtest/test_retain_pvc.py

```diff
@@ -0,0 +1,17 @@
+from ocs_ci.framework.testlib import libtest
+from ocs_ci.ocs import constants, resources
+
+
+@libtest
+def test_main(teardown_factory, pvc_factory):
+    sc = resources.ocs.OCS(
+        kind=constants.STORAGECLASS,
+        metadata={'name': 'ocs-storagecluster-ceph-rbd'}
```

Comment:
@fbalak Sorry I got confused, why only RBD strageclass here ?

Reply:
This is just a libtest that is not a part of a test suite that is run periodically (tests with marker @libtest are not run in CI). I created this test for us to make sure that we teardown PVCs with Retain Reclaim Policy correctly.